gpytorch/__init__.py

```diff
@@ -119,7 +119,41 @@ def mvn_kl_divergence(mean_1, chol_covar_1, mean_2, covar_2):
 
     For more information, see the MVNKLDivergence function documentation.
     """
-    return MVNKLDivergence()(mean_1, chol_covar_1, mean_2, covar_2)
+    if isinstance(covar_2, LazyVariable):
+        return covar_2.mvn_kl_divergence(mean_1, chol_covar_1, mean_2)
+    else:
+        return MVNKLDivergence()(mean_1, chol_covar_1, mean_2, covar_2)
+
+
+def add_jitter(covar):
+    if isinstance(covar, LazyVariable):
+        covar.jitter_()
+    else:
+        covar.data.add_(1e-3 * torch.eye(len(covar)))
+
+
+def initialize_variational_parameters(var_output):
```

Comment:
Does this belong in `gpytorch.__init__`? This file is getting super bloated, and this doesn't seem like a top-level function.